build/distributions/CentOS/7/x86_64/yum_data/packages.yaml

```diff
@@ -21,32 +21,32 @@ chkrootkit:
   :rpm_name: chkrootkit-0.50-4el7.x86_64.rpm
   :source: https://packagecloud.io/simp-project/6_X_Dependencies/packages/el/7/chkrootkit-0.50-4el7.x86_64.rpm
 clamav:
-  :rpm_name: clamav-0.101.1-1.el7.x86_64.rpm
-  :source: http://lug.mtu.edu/epel/7/x86_64/Packages/c/clamav-0.101.1-1.el7.x86_64.rpm
+  :rpm_name: clamav-0.101.0-1.el7.x86_64.rpm
```

Comment:
simp-core won't build with these reverted.  The packages are no longer available.  Also, IMHO, we should always use the latest antivirus packages.